js/console.ts

```diff
@@ -473,4 +516,9 @@ export class Console {
 
     this.info(`${label}: ${duration}ms`);
   };
+
+  clear = (): void => {
+    cursorTo(stdout, 0, 0);
+    clearScreenDown(stdout);
```

Comment:
Is this what Node does?

Reply:
Yes, it is. I used as reference.

Reply:
maybe we should implement `group*` first and then implement this `clear`.

Node.js does NOT fully implement the features in the specification.
 
follow the [specification](https://console.spec.whatwg.org/#clear)

> 1.1.2. clear()**
> 
> 1. **Empty the appropriate [group stack](https://console.spec.whatwg.org/#group-stack)**.
> 1. If possible for the environment, clear the console. (Otherwise, do nothing.)

I tested in Chrome 73.0.3642.0, Firefox 66.0a1, Edge 17.1713 on Win10, both of all clear the group-stack and console. Node.js only clear the console.

Reply:
@justjavac 
Thanks for review. Well then, I'll be working on the `group*`. Is this all right?